pkgs/servers/http/dufs/default.nix

```diff
@@ -23,6 +23,10 @@ rustPlatform.buildRustPackage rec {
     Security
   ];
 
+  checkFlags = [
```

Comment:
do you mind adding some explanation? it already builds on linux and this doesn't seem to fix the build on darwin